cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -773,7 +773,7 @@ func buildScaleDownStatusClusterwide(candidates map[string][]string, lastProbed
 func isNodeNotStarted(node *apiv1.Node) bool {
```

Comment:
nit: If you are changing this part of code consider removing Not from name of this function.
Maybe call it "isNodeStillStarting" or "isNodeStarted" negating the semantics.

Reply:
Done